consensus/pacemaker/module.go

```diff
@@ -51,7 +52,7 @@ type pacemaker struct {
 	// Only used for development and debugging.
 	debug pacemakerDebug
 
-	// REFACTOR: this should be removed, when we build a shared and proper logger
```

Comment:
Can you add the `Refactor` back? We should be able to remove `logPrefix` in a future commit